client/src/app/(protected)/home/page.tsx

```diff
@@ -31,7 +31,7 @@ export default async function HomePage() {
               <li className='mt-3'>
                 {/* this is a temporary button to test functionality easier */}
                 <Link
-                  href='/createGroup'
+                  href='/group/create'
```

Comment:
```suggestion
                  href='/groups/new'
```

Nit: El `new` generalmente se usa para renderizar el Form para crear un recurso, y el `create` termina siendo el POST que se hace luego del `submit`.


No nit: Lo que sí me parece importante es lo de pluralizar las URLs.

`GET /groups` -> trae todos los groups
`GET /groups/:id` -> trae el grupo con ese ID
`GET /groups/new` -> renderiza el form para crear uno nuevo
`POST /groups` -> crea uno nuevo
`GET /groups/edit` -> renderiza el form para editar un group
`PATCH /groups/:id` -> updatea el grupo con ese ID
Etc...